gamemodes/hunters_glee/gamemode/sh_shopitems.lua

```diff
@@ -3013,6 +3027,22 @@ local defaultItems = {
         purchaseCheck = unUndeadCheck,
         purchaseFunc = nailerPurchase,
     },
+    [ "frag" ] = {
+        name = "Grenades",
+        desc = "Grenade \nSimple explosives, useful for hordes!",
+        cost = 45,
+        markup = 2.5,
+        markupPerPurchase = 0.45,
```

Comment:
```suggestion
        cost = 75,
        markup = 2,
        markupPerPurchase = 0.25,
```

Make it more expensive, but rampup cost slower